src/tests/ftest/security/cont_overwrite_acl.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. 8F-30005.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import os
+
+from cont_security_test_base import ContSecurityTestBase
+from security_test_base import create_acl_file
+from command_utils import CommandFailure
+from avocado.utils import process
+from avocado import fail_on
+
+
+class OverwriteContainerACLTest(ContSecurityTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description:
+
+    Test to verify ACL entry overwrite.
+
+    :avocado: recursive
+    """
+    def setUp(self):
+        """Set up each test case."""
+        super(OverwriteContainerACLTest, self).setUp()
+        self.daos_cmd = self.get_daos_command()
+        self.prepare_pool()
+        self.add_container(self.pool)
+
+    def test_acl_overwrite_invalid_inputs(self):
+        """
+        JIRA ID: DAOS-3708
+
+        Test Description: Test that container overwrite command performs as
+            expected with invalid inputs in command line and within ACL file
+            provided.
+
+        :avocado: tags=all,pr,security,container_acl,cont_overwrite_acl
```

Comment:
Is it normal to use the same tag, cont_overwrite_acl in this case, for every test case? I thought it would be more helpful to use different tag for different test case so that we can run test individually in CI, or maybe my understanding is wrong.

Reply:
Yes, thanks for pointing that out, I overlooked it.